natsjobs/config.go

```diff
@@ -21,7 +21,7 @@ type config struct {
 
 	Priority           int64  `mapstructure:"priority"`
 	Subject            string `mapstructure:"subject"`
-	Stream             string `mapstructure:"stream"`
+	StreamID           string `mapstructure:"stream_id"`
```

Comment:
Unfortunately, this is BC, we can't change this key. We can btw use the updated key in the driver, but not in the config :cry:

Reply:
You can leave `StreamID` field, but use `mapstructure:"stream"` for the field option.